### PR TITLE
Rename to pgrx

### PR DESCRIPTION
**Pull request by eeeebbbbrrrr**
This does the needful behind issue #1106

### CODE REVIEW

**Review by workingjubilee — APPROVED** [review]
We discussed the article question, which was the only serious question I had.

**Comment by eeeebbbbrrrr** [comment]
> We discussed the article question, which was the only serious question I had.

yeah, I think it's fine for the files in `./articles/` to get the rename treatment too, since they're under source control.

Published versions on corporate blog can be looked at later, or even never.

File: .github/docker/Dockerfile.alpine:3.16 [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
+# Example of how to build this container (ran from pgrx checkout directory):
+#   docker build --build-arg PG_MAJOR_VER=14 -f .github/docker/Dockerfile.alpine\:3.16 -t pgrx-alpine .
 #
 # Example of how to run this container with tests after building the image:
-#   docker run pgx-alpine cargo test --no-default-features --features pg14 --locked
+#   docker run pgrx-alpine cargo test --no-default-features --features pg14 --locked
 #
 # Note that "PG_MAJOR_VER" build arg in the build step must match the
-# "--features pgXX" in the run step
+# "--features pgrxX" in the run step
```
Comment (syvb) [inline .github/docker/Dockerfile.alpine:3.16:8]: This (and all the other instances of `pgXX`) shouldn't have been changed

Comment (eeeebbbbrrrr) [reply .github/docker/Dockerfile.alpine:3.16:8]: nice catch.  thanks.  we're still reviewing this too.

**Comment by workingjubilee** [comment]
Yeah, that seems fine, and it definitely seems lower priority now that I realize the tcdioss website won't automatically update.

Maybe we should open an issue to track following up on that and making a final decision. But "we discussed it and made a decision that will suffice for now" was the main thing I wanted.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -98,10 +98,10 @@ Do this *in order*:
...
-You agree that all code you submit in pull requests to https://github.com/tcdi/pgx/pulls
+You agree that all code you submit in pull requests to https://github.com/tcdi/pgrx/pulls
 is offered according to the MIT License, thus may be freely relicensed and sublicensed,
 and that you are satisfied with the existing copyright notice as of opening your PR.
 
 [crates.io]: https://crates.io
-[pgx@crates.io]: https://crates.io/crates/pgxa
-[pgx@docs.rs]: https://docs.rs/pgx/latest/pgx
\ No newline at end of file
+[pgrx@crates.io]: https://crates.io/crates/pgrxa
```
Comment (eeeebbbbrrrr) [inline CONTRIBUTING.md:106]: this link was wrong in the orignal too.  we'll fix it in post.

**Comment by eeeebbbbrrrr** [comment]
Alrighty, this is going to master right now as a squash merge, then merged into develop, then the repo here on GitHub will be renamed.  Then published to crates.io from the master branch.

**Pull request merged**